components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -231,7 +229,7 @@ impl Recorder {
         split_balance_score: f64,
         split_contained_score: f64,
         sample_threshold: i32,
-    ) -> Vec<u8> {
+    ) -> Option<Vec<u8>> {
```

Comment:
Why use `Option`?

Reply:
to distinguish between vec![] and empty key

Reply:
Why need to distinguish both? Split key can't be `vec![]`.